webapp/src/websocket/index.ts

```diff
@@ -14,3 +14,10 @@ export function handleDisconnect(store: Store<GlobalState, Action<Record<string,
         store.dispatch(toggleIsConnected(false) as Action);
     };
 }
+
+// TODO: types are already added in a PR so, update that here when synced with that
```

Comment:
```
// TODO: Types are already added in a PR so, update that here when synced with that
```

Reply:
this comment meant for removal at the end so, I don't think rephrasing this makes any use